exercises/minesweeper/description.md

```diff
@@ -1,25 +1,35 @@
-Add the numbers to a minesweeper board.
+Add the score totals to a completed Minesweeper board.
 
 Minesweeper is a popular game where the user has to find the mines using
 numeric hints that indicate how many mines are directly adjacent
 (horizontally, vertically, diagonally) to a square.
 
 In this exercise you have to create some code that counts the number of
-mines adjacent to a square and transforms boards like this (where `*`
-indicates a mine):
-
-    +-----+
-    | * * |
-    |  *  |
-    |  *  |
-    |     |
-    +-----+
-
-into this:
-
-    +-----+
-    |1*3*1|
-    |13*31|
-    | 2*2 |
-    | 111 |
-    +-----+
+mines adjacent to a given empty square and replaces that square with the
+count.
+
+The board is a rectangle composed of blank space (' ') characters. A mine 
+is represented by an asterix ('\*') character.
```

Comment:
I see that the character was escaped for the purposes of Markdown (you can see when viewing a rendered version of this file that the backslash isn't there). I have a reservation that this disadvantages those individuals who would view this file with a text editor instead of a Markdown renderer since it may otherwise take them some time to understand what the backslash is for.
(P.S. "those individuals" includes me) 

I hope there is a way to do this that is friendly both to readers using a text editor and to readers using a Markdown renderer.

If it turns out the former group does not make up a significant population and therefore their concerns are invalid, feel free to carry on merrily without regard to this comment.

Reply:
The file has a .md extension, hence we're going to have to assume that the reader understands markdown conventions (otherwise we can't use triple tildes for code blocks, etc). Unfortunately there's no easy way to be friendly to plain text rendering without treating the blank space differently than the asterisk.

Reply:
Fixed.